Dockerfile

```diff
@@ -61,9 +61,6 @@ COPY --from=builder /opt/app-root/src/entrypoint/entrypoint ${USER_HOME}/bin/ent
 # Set operator bin owner and permissions
 RUN chown -R `id -u`:0 ${USER_HOME}/bin && chmod -R 755 ${USER_HOME}/bin
 
-# Upgrade packages
-RUN dnf update -y --setopt=install_weak_deps=0 && rm -rf /var/cache/yum
```

Comment:
I guess you are removing this line because dnf is not available on ubi8-minimal. If this is the only reason you can replace dnf with microdnf, see https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/building_running_and_managing_containers/assembly_types-of-container-images_building-running-and-managing-containers#con_understanding-the-ubi-init-images_assembly_types-of-container-images

Reply:
@brusdev Good call. Updated.